dali/pipeline/operators/displacement/displacement_filter.h

```diff
@@ -42,6 +42,14 @@ struct HasParam <T, decltype((void) (typename T::Param()), 0)> : std::true_type
 template <typename T>
 struct Point {
   T x, y;
+  template <typename U>
+  Point<U> Cast() {
+    return {static_cast<U>(x), static_cast<U>(y)};
+  }
+  template <typename U>
+  Point<T> operator-(Point<U> r) {
```

Comment:
This arithmetic is incompliant with C++ regular type policy. For example:
```
int a=5;
float b=3.2;
auto c = a-b; // decltype(c) == float
```
However, here:
```
Point<int> a = {4,2};
Point<float> b = {5,2};
auto c = a-b; // decltype(c) == Point<int>
```
In my opinion it should at least generate some warning or sth

Reply:
You're right, removed it.